src/_pytest/nodes.py

```diff
@@ -127,6 +127,20 @@ def _imply_path(
 
 
 class NodeMeta(type):
```

Comment:
@nicoddemus mentioned elsewhere that _ indicates private but also that a private access modifier is implied when already inside an internal module. 

As an implementation detail, I'm wondering if `NodeMeta` should be so designated: `_NodeMeta`.
Or is it already implied by its parent directory `_pytest`?

Reply:
its already implied by _pytest